src/home/components/DefaultHomeCard.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright © 2021-2023 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+import React from 'react';
+
+import {
+  Box,
+  Button,
+  Divider,
+  Link,
+  Paper,
+  Stack,
+  Typography,
+} from '@mui/material';
+
+import CardActionRouterLink from 'common/components/CardActionRouterLink';
+import RouterLink from 'common/components/RouterLink';
+
+import HomeCard from './HomeCard';
+
+const DefaultHomeCard = ({
+  item,
+  title,
+  link,
+  url,
+  pictureUrl,
+  image,
+  alt,
+  children,
+  showActionsAsButtons = false,
+  backgroundColor = '#F5F7FC',
```

Comment:
Issue: We should always use the colors from the palette, if the color is not there a new one should be added there (See: https://github.com/techmatters/terraso-web-client/blob/main/src/theme.js#L34). For example: 
```suggestion
  backgroundColor = 'gray.lite2',
```